src/cephadm/cephadm

```diff
@@ -1786,6 +1786,10 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
         if daemon_type == 'osd':
             # osds have a pre-start step
             assert osd_fsid
+            if osd_simple:
+                for n in ['block', 'block.db', 'block.wal']:
+                    p = os.path.join(data_dir, n)
+                    f.write('[ -L {p} ] && chown {uid}:{gid} {p}\n'.format(p=p, uid=uid, gid=gid))
```

Comment:
```suggestion
                    code = """
# This is a simple OSDs. Needs chown on startup:
[ -L {p} ] && chown {uid}:{gid} {p}\n""".format(p=p, uid=uid, gid=gid)
                    f.write(code)
```
?

Reply:
Good idea, although that will put the comment before each of the three chown lines, e.g.:

```
# This is a simple OSDs. Needs chown on startup:
[ -L /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block ] && chown 167:167 /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block
# This is a simple OSDs. Needs chown on startup:
[ -L /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block.db ] && chown 167:167 /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block.db
# This is a simple OSDs. Needs chown on startup:
[ -L /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block.wal ] && chown 167:167 /var/lib/ceph/f6cf67cc-dcfa-3de8-a33f-cab60875412f/osd.21/block.wal
```

We could alternately do:

```diff --git a/src/cephadm/cephadm b/src/cephadm/cephadm
index 0652d89b280..18d1b30eba1 100755
--- a/src/cephadm/cephadm
+++ b/src/cephadm/cephadm
@@ -1787,6 +1787,7 @@ def deploy_daemon_units(fsid, uid, gid, daemon_type, daemon_id, c,
             # osds have a pre-start step
             assert osd_fsid
             if osd_simple:
+                f.write('# This is a simple OSD. Needs chown on startup:\n')
                 for n in ['block', 'block.db', 'block.wal']:
                     p = os.path.join(data_dir, n)
                     f.write('[ -L {p} ] && chown {uid}:{gid} {p}\n'.format(p=p, uid=uid, gid=gid))
```

...which would put the comment only once at the start.

Thoughts?

Reply:
much bettter!

Reply:
Done!